caom2-artifact-resolvers/src/main/java/ca/nrc/cadc/caom2/artifact/resolvers/CadcMastResolver.java

```diff
@@ -94,8 +94,20 @@
     public static final String SCHEME = "mast";
     private static final Logger log = Logger.getLogger(CadcMastResolver.class);
     private static final URI DATA_RESOURCE_ID = URI.create("ivo://cadc.nrc.ca/data");
-    private String baseDataURL;
+    private static final String HST_ARCHIVE = "HST";
+    
+    // For HST archive only, 
+    // backwards compatible is MAST/HST/product/<filename>
+    // not backwards compatible is HST/product/<filename>
+    private boolean isBackwardsCompatible = true;
```

Comment:
Do we need backwards compatibility in the resolver?  I thought the data web service was the only thing needing to support backwards compatible GETs.

Reply:
This change was done at the beginning of story 2601. There have been a lot of decision changes. However the changes are basically still applicable to story 2591 which is related to moving HST files from MAST archive to the new HST archive. CadcMastResolver is not used by story 2601; however it will be need by story 2591. Since the changes have been made in branch s2601, I thought I shall just push the changes now. CadcMastResolver will be removed after we are done with 2591. Testing so far has been done with CadcMastResolver included.